ci-operator/jobs/openshift/installer/openshift-installer-presubmits.yaml

```diff
@@ -62,6 +62,8 @@ presubmits:
             configMapKeyRef:
               key: master.json
               name: ci-operator-openshift-installer
+        - name: BAZEL_IMAGE
+          value: quay.io/coreos/tectonic-builder:bazel-v0.3
         image: ci-operator:latest
```

Comment:
This is only to get the bazel binary available, to run `bazel build smoke_tests` .  If we can add that to the base image, we  won't need this.